B827EBFFFE65CF5D

```diff
@@ -0,0 +1,18 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "B827EBFFFE65CF5D ",
```

Comment:
you have an extra space between the ID and the last quote that will break the file. Please fix and then we merge.

Reply:
fixed !